src/components/Filters/RangeInput.tsx

```diff
@@ -153,9 +153,18 @@ export function RangeInput(props: RangeInputProps): JSX.Element | null {
   }, [answersActions, fieldId, getFilterDisplayName, isValid, rangeFilter]);
 
   const handleClickClear = useCallback(() => {
+    const displayName = getFilterDisplayName(rangeFilter.value);
+    clearStaticRangeFilters(answersActions, new Set([fieldId]));
+    answersActions.setFilterOption({
+      ...rangeFilter,
+      selected: false,
+      displayName
+    });
```

Comment:
I tested them and it worked with using either one. Can I use setFilterOption instead of clearStaticRangeFilters? I feel like it would be easier to test that the corresponding filter has not been set in state..

Reply:
I think using `setFilterOption` makes sense for the button `clear min-max` specific to this filter group.